README.md

```diff
@@ -34,19 +34,16 @@ You should have received a copy of the GNU General Public License
 along with Damn Vulnerable Web Application (DVWA).  If not, see http://www.gnu.org/licenses/.
 
 - - -
-## Download and install as a docker container
-- [dockerhub page](https://hub.docker.com/r/vulnerables/web-dvwa/)
-`docker run --rm -it -p 80:80 vulnerables/web-dvwa`
-
-Please ensure you are using aufs due to previous MySQL issues. Run `docker info` to check your storage driver. If it isn't aufs, please change it as such. There are guides for each operating system on how to do that, but they're quite different so we won't cover that here.
 
 ## Download
 
-DVWA is available either as a package that will run on your own web server or as a Live CD:
+While there are various versions of DVWA around, the only supported version is the latest source from the offical Github repository. You can either lone it from the repo:
+

```

Comment:
Typo `lone` should be `clone`

Reply:
The GitHub built in markdown editor doesn't do spell checking, that's annoying. I've fixed a few typos.